### ISSUE TITLE
Warning with `TEMPLATE_CONTEXT_PROCESSORS` and Django 1.11

### ISSUE THREAD

**Issue by Postroutine** [rossp/django-menu#26]
When I set `TEMPLATE_CONTEXT_PROCESSORS` in the `settings.py`, I got a warning from Django:

```
WARNINGS:
?: (1_8.W001) The standalone TEMPLATE_* settings were deprecated in Django 1.8 and the TEMPLATES dictionary takes precedence. You must put the values of the following settings into your default TEMPLATES dict: TEMPLATE_CONTEXT_PROCESSORS
```

**Comment by rossp** [comment]
The documentation will need to be updated for Django 1.8+ template settings.

**Comment by miladazadsanjary** [comment]
when id add   'django.core.context_processors.request', in template context processors i deal with this error :  
Exception Value: | No module named 'django.core.context_processors'
in django 1.11 
is there any solution for this problem ?

**Comment by burbilog** [comment]
This one works in 1.11.12:
```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates')],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
                'django.template.context_processors.request',
            ],
        },
    },
]
```
Documentation should be updated, there is no TEMPLATE_CONTEXT_PROCESSORS anymore.

**Comment by rossp** [comment]
Fixed in 2c3964181cdb3d2008c464d41b6f881381e98568.
